diff -Naur Python-2.7.16-orig/setup.py Python-2.7.16/setup.py
--- Python-2.7.16-orig/setup.py	2019-03-02 19:17:42.000000000 +0100
+++ Python-2.7.16/setup.py	2019-03-04 17:14:48.436500356 +0100
@@ -497,7 +497,8 @@
                         add_dir_to_list(dir_list, directory)
 
         if os.path.normpath(sys.prefix) != '/usr' \
-                and not sysconfig.get_config_var('PYTHONFRAMEWORK'):
+                and not sysconfig.get_config_var('PYTHONFRAMEWORK') \
+                and not "DONT_ADD_INCLUDEDIR_AND_LIBDIR" in os.environ:
             # OSX note: Don't add LIBDIR and INCLUDEDIR to building a framework
             # (PYTHONFRAMEWORK is set) to avoid # linking problems when
             # building a framework with different architectures than
